builder/xenserver/common/step_export.go

```diff
@@ -93,6 +93,42 @@ func (StepExport) Run(state multistep.StateBag) multistep.StepAction {
 		return multistep.ActionHalt
 	}
 
+	if len(config.FinalNetworkNames) > 0 {
+		vifs := state.Get("vifs").([]*xsclient.VIF)
```

Comment:
The other way of doing this would be to call out to XAPI and ask for the list of VIFs, and then remove each of those (if a template is cloned originally, there could be more VIFs (?)) but this is probably fine.